openmsupply/open-msupply
# 1950 endpoint insert prescription invoice line

Fixes #1950 

# 👩🏻‍💻 What does this PR do? 
Endpoint to insert a prescription invoice.

# 🧪 How has/should this change been tested? 
- Run tests.
- Call endpoint to make sure it's working, but must have a prescription invoice first.
